nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -14,6 +14,20 @@ These release notes track the development of the {run-once-operator} for {produc
 
 For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
 
+[id="run-once-duration-override-operator-release-notes-1-1-2"]
+== Run Once Duration Override Operator 1.1.2
+
+Issued: 31 October 2024
+
+The following advisory is available for the {run-once-operator} 1.1.2:
+
+* link:https://access.redhat.com/errata/RHSA-2024:8337[RHSA-2024:8337]
+
+[id="run-once-duration-override-operator-1-1-2-bug-fixes"]
+=== Bug fixes
+
+* This release of the {run-once-operator} addresses several Common Vulnerabilities and Exposures (CVEs).
+
 [id="run-once-duration-override-operator-release-notes-1-1-0"]
```

Comment:
This is skipping right from 1.1.0 to 1.1.2.

Per https://github.com/openshift/run-once-duration-override-operator?tab=readme-ov-file#releases, 1.1.1 should also be present on 4.15. I think you need to add that here in 4.15 (unfortunately will need CM probably)